GFL DK Weapon Changes Proposal.md

```diff
@@ -0,0 +1,171 @@
+
+Shotguns: Extreme powerhouses at close range and falters hard at long. Vanilla shotguns rely on the sheer number of pellets to take down armored enemy dudes by having each pellet roll against the armor coverage of the target… eventually some makes its way through and enough will kill, even at long range. My change to this is nearly zero aside from tuning the damage a bit higher at closer ranges and lowering it at longer ranges so hopefully we dont get absurd things like Vepley sniping some dude across the map in two shots.
+
+Sniper Rifles: The opposite of the shotgun. Sniper rifles dont exist in vanilla so I am just going to reference your stuff for Springfield, Makiatto and Nemesis. Basically since most of these snipers are suffering from low ROF and magazine size already, we will give them a substantial buff in accuracy and aim time at close range but will still make their best at long range!
+
+LMGs: Basically heavier assault rifles with the ability to suppress, meaning their aim times are a bit slower and their ready and guard times will be slower as well. Mind that the suppression technique will not be limited to them, as having teams without LMGs such as Groza Squad, DEFY and Zucchero would need one rapid fire weapon to fill for them
+
+Melee weapons dont need much adjustment… Krolik is as fine as it is.
+
+
+So you noticed no mention of shields yes? Well I have this plan where I will do my best to make new animation sets for specific T-Dolls I see fit for frontlining based from whatever existing official characterization they have in the GFL universe. Take note that unless otherwise mentioned, the shielders will be able to use their Signature Weapons with the shield.
+
+Here are the dolls I see fit to wield shields from every team:
+
+Groza Squad: Groza and Colphne (Colphne being small and no mention of massive strength, gets really crippling penalties with a shield)
+
+Note: (Krolik can't be the shielder because 1 I dont know if i can mesh melee with shields and 2 Krolik’s shield does jackshit ingame, her defenses are very low.)
+
+DEFY: Alva and Voymastina (Alva wont be able to use her AN94 with the shield, Voymastina gets lesser penalties with the shield because she is strong…)
+
+Zucchero: Centaureissi and Springfield (Springfield will not be able to use her springfield with a shield, limiting her to backup pistols)
+
+H.I.D.E. 404: Andoris and Leva (Leva gets to have the shield to help her abysmal ROF weapon to be useful when teamed up with Lenna, Android meanwhile is the tank of the team in universe, always the careful one.)
```

Comment:
why is leva's ROF low ? the UMP45 and UMP9 should have a similar rof (around 650 bullets per minute). 

the difference between them should be that ump45 fires .45 ACP rounds with more stopping power but lower magazine capacity (25 rounds), while the ump9 fires 9mm rounds with less recoil, higher capacity
  (30 rounds), and better penetration.

although, again this is only if we're going for the realism angle 

but the original question still remains - why does the ump45 have a lower RoF ? im guessing it has to do with the ammo they're using (why DK2 puts rate of fire in ammo, I don't understand)